framework/include/cppmicroservices/BundleContext.h

```diff
@@ -545,9 +545,18 @@ class US_Framework_EXPORT BundleContext
    * associated with this <code>reference</code> has been unregistered.
    *
    * <p>
+   * The <code>ServiceObjects</code> object must be used to obtain multiple service objects for
+   * services with prototype scope. For services with singleton or bundle scope,
+   * the ServiceObjects::GetService() method behaves the same as the
+   * GetService(const ServiceReference<S>&) method and the 
+   * ServiceObjects::UngetService(const ServiceReferenceBase&) method behaves the
```

Comment:
It looks like this excerpt was copied from the an outdated documentation block of the ServiceObjects class. There is no `UngetService(const ServiceReferenceBase&)` function any more. I suggest to just delete this part here and from the ServiceObjects class documentation.

Reply:
will do.